src/database.cpp

```diff
@@ -211,10 +215,18 @@ std::string Database::open(bool wait, bool memory) {
     sqlite3_free(fail);
 
     if (!wait || ret != SQLITE_BUSY) {
+      if (waiting) {
+        std::cerr << std::endl;
+      }
       close();
       return out;
```

Comment:
I'm not sure when this branch is executed but I added it anyway 🤷

Reply:
@terpstra does this one need to be here? I'm not sure when we're returning vs. breaking from the loop.